solr/solrj/src/java/org/apache/solr/common/params/CommonParams.java

```diff
@@ -155,6 +155,18 @@
   String SEGMENT_TERMINATE_EARLY = "segmentTerminateEarly";
   boolean SEGMENT_TERMINATE_EARLY_DEFAULT = false;
 
+  /**
+   * Default parser for parsing filter queries ({@code fq}.
+   */
+  String FQ_QPARSER = CommonParams.FQ + ".qparser";
+
+  /**
+   * Default parser for parsing filter queries ({@code fq}. This is an
+   * alias to {@link #FQ_QPARSER} which is aligned with the legacy {@code defType}
+   * parameter. {@link #FQ_QPARSER} should be preferred over this syntax.
+   */
+  String FQ_DEFPARSER = CommonParams.FQ + ".defType";
```

Comment:
I don't see the point in introducing fq.defType if we agree fq.qparser makes more sense?

Reply:
Thanks. Yeah, I didn't think of it - I'll take a look at those other places. I don't mind adding just one version of the parameter too (fq.qparser).